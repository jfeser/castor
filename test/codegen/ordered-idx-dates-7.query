      AOrderedIdx(OrderBy([f1 asc, f2 asc], Dedup(Select([f as f1, f as f2], r_date))) as k,
           AScalar(k.f1 as v), >= date("0000-01-01"), <= date("9999-01-01"), >= date("0000-01-01"), < date("2018-09-01"))
