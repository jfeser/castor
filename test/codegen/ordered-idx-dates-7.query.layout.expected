0:25 Ordered idx map
0:4 Ordered idx key
0:4 Tuple body
0:2 Scalar (=(Date 2016-12-01))
0:0 Ordered idx index len (=25)
0:0 Ordered idx len (=35)
0:0 Tuple len (=4)
2:2 Scalar (=(Date 2016-12-01))
4:1 Ordered idx ptr (=0)
5:4 Ordered idx key
5:4 Tuple body
5:2 Scalar (=(Date 2017-10-05))
5:0 Tuple len (=4)
7:2 Scalar (=(Date 2017-10-05))
9:1 Ordered idx ptr (=2)
10:4 Ordered idx key
10:4 Tuple body
10:2 Scalar (=(Date 2018-01-01))
10:0 Tuple len (=4)
12:2 Scalar (=(Date 2018-01-01))
14:1 Ordered idx ptr (=4)
15:4 Ordered idx key
15:4 Tuple body
15:2 Scalar (=(Date 2018-01-23))
15:0 Tuple len (=4)
17:2 Scalar (=(Date 2018-01-23))
19:1 Ordered idx ptr (=6)
20:4 Ordered idx key
20:4 Tuple body
20:2 Scalar (=(Date 2018-09-01))
20:0 Tuple len (=4)
22:2 Scalar (=(Date 2018-09-01))
24:1 Ordered idx ptr (=8)
25:10 Ordered idx body
25:2 Scalar (=(Date 2016-12-01))
27:2 Scalar (=(Date 2017-10-05))
29:2 Scalar (=(Date 2018-01-01))
31:2 Scalar (=(Date 2018-01-23))
33:2 Scalar (=(Date 2018-09-01))
